### ISSUE TITLE
SP5 ScatterPlot: Implement StepDisplay for ScatterPlot

### ISSUE THREAD

**Issue by NoahSigl** [ScottPlot/ScottPlot#3234]
**Suggestion:** Bring back StepDisplay

I am thinking about upgrading to ScottPlot 5.0 in my project.
The only reason i did not do it yet is the missing StepDisplay which is essential in my software.
Is it possible to add this feature back in ScottPlot 5.0?

My previous code looked like this for which i didn't find an alternative for:

```C
var sp = WpfPlot.Plot.Add.Scatter(dateDoubles, valueArray, color);

sp.StepDisplay = true;
sp.StepDisplayRight = true;
```

If it's still in SP5 and i just didn't find the alternative please let me know and my apologies for bringing it up.
Thank you

**Comment by swharden** [comment]
Hi @NoahSigl, thanks for this suggestion! I definitely want to get this added eventually. However there are some higher priority issues I'm focusing on for the next several days, so if you or someone else wants to add this feature to ScottPlot 5 and create a pull request, I'd be happy to merge it!

Here's the related code from ScottPlot 4 you can mimic:

https://github.com/ScottPlot/ScottPlot/blob/958df748f7110d27a15836250e1b20fac16b6a72/src/ScottPlot4/ScottPlot/Plottable/ScatterPlot.cs#L463-L475

https://github.com/ScottPlot/ScottPlot/blob/958df748f7110d27a15836250e1b20fac16b6a72/src/ScottPlot4/ScottPlot/Plottable/ScatterPlot.cs#L515-L531

**Comment by swharden** [comment]
resolved in #3242
